lua/lspconfig/ui/windows.lua

```diff
@@ -19,13 +21,18 @@ end
 
 local win_float = {}
 
-win_float.default_options = {
-  winblend = 15,
-  percentage = 0.9,
+win_float.default_options = {}
+
+local _mt = {
+  __newindex = function(t, k, v)
+    rawset(t, k, v)
+  end,
```

Comment:
This is what assigning a new index already does by default. Any reason why a metatable is used for `default_options`?